lighthouse-core/test/report/html/renderer/crc-details-renderer-test.js

```diff
@@ -23,7 +23,6 @@ const superLongURL =
     'https://example.com/thisIsASuperLongURLThatWillTriggerFilenameTruncationWhichWeWantToTest.js';
 const DETAILS = {
   type: 'criticalrequestchain',
-  header: {type: 'text', text: 'CRC Header'},
```

Comment:
only executable code change (though in a test). We haven't done anything with a `header` field in I don't know how long, though, so might as well delete it today :)